src/jit/alloc.cpp

```diff
@@ -115,16 +111,16 @@ ArenaAllocator& ArenaAllocator::operator=(ArenaAllocator&& other)
     assert(!isInitialized());
 
     m_memoryManager = other.m_memoryManager;
-    m_firstPage = other.m_firstPage;
-    m_lastPage = other.m_lastPage;
-    m_nextFreeByte = other.m_nextFreeByte;
-    m_lastFreeByte = other.m_lastFreeByte;
+    m_firstPage     = other.m_firstPage;
```

Comment:
Hrm, I take it that we'll keep the crazy vertical alignment that's already everywhere and make it even worse after formatting...